DataFormats/Detectors/FIT/FV0/src/RawEventData.cxx

```diff
@@ -0,0 +1,110 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "DataFormatsFV0/RawEventData.h"
+#include <CommonDataFormat/InteractionRecord.h>
+#include <Framework/Logger.h>
+#include <iostream>
+
+using namespace o2::fv0;
+
+using namespace std;
+
+ClassImp(RawEventData);
```

Comment:
just for your info: this is not necessary anymore (though does not hide either)

Reply:
I cleaned it up